default/cve5/portal.js

```
@@ -654,6 +657,24 @@ function transatePath(p) {
+    // KSF
+    var j = await mainTabGroup.getValue();
+    res = await fetch('/publishcve', {
+        method: 'POST',
+        credentials: 'include',
+        headers: {
+            'Accept': 'application/json, text/plain, */*',
+            'Content-Type': 'application/json',
+            'X-CSRF-Token': csrfToken,
+        },
+        redirect: 'error',
+        body: JSON.stringify({cve:j.cveMetadata.cveId})
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)